tests/serde-se.rs

```diff
@@ -547,6 +547,397 @@ mod without_root {
                     </Untagged>");
         }
     }
+
+    mod with_indent {
+        use super::*;
+        use pretty_assertions::assert_eq;
+
+        macro_rules! serialize_as {
+            ($name:ident: $data:expr => $expected:literal) => {
+                #[test]
+                fn $name() {
+                    let mut ser = Serializer::new(String::new());
+                    ser.indent(' ', 2);
```

Comment:
Expected output for other tests is also indented, despite not having an indentation set?  One example: https://github.com/tafia/quick-xml/pull/490/files#diff-452d3372c13dbffe2706a379e9b64cb6d109a116360ca59ea6efe3ee8c403b08R411

I'm confused whether those tests ought to not expect any indentation.

Reply:
FWIW: For the tests expecting proper indentation, I've found [indoc](https://docs.rs/indoc/latest/indoc/) to be useful. It strips leading spaces up to the indentation point but no further, e.g.:
```rust
indoc! {r#"
    <foo>
        <bar>42</bar>
    </foo>"# }
```

becomes
```rust
<foo>
    <bar>42</bar>
</foo>
```